etc/ci/update_nightlies.sh

```diff
@@ -62,14 +62,24 @@ MASTER_COMMIT="$(git rev-parse HEAD)"
 
 git checkout -b gh-pages upstream/gh-pages || exit 1
 
-git rm -rf nightly || true
-rm -rf nightly || true
-
-mkdir nightly
+mkdir -p nightly
+
+# Delete files in /nightly not having a commit hash younger than 1 day
+# (The saunders redirects are only updated once a day.)
+YESTERDAY="$(date -d '1 day ago' +%s)"
+find nightly -type f | while read nfile
+do
+    HASH="$(echo $nfile | sed -r 's,^.*-([0-9a-z]+).pdf$,\1,')"
+    FILEDATE="$(git show -s --format=%ct $HASH 2>/dev/null)"
+    if [ -z $FILEDATE ] || [ $YESTERDAY -ge $FILEDATE ];
```

Comment:
I would suggest quoting these variables (`$nfile`, `$HASH`, `$FILEDATE`, `$YESTERDAY`), to be on the safe side ("always quote every variable unless you have an explicit reason not to"), but it probably doesn't matter much.